### PR TITLE
bugfix for cleaning /storage

### PR DESCRIPTION
**Pull request by OlivierBondu**
The deletion of stuff on /storage was potentially incorrect: the `cleanOldFrameworkTasksNotInDB.py` was relying on the sample path, which could differ from the list of files actually listed by the DB entry

Unfortunately I did ran the script before realizing this fix may be needed so I cannot test any longer...

### CODE REVIEW

File: scripts/cleanOldFrameworkTasksNotInDB.py [hunk 1]
```diff
@@ -56,10 +57,17 @@ def main(crabUsername, ingridUsername):
     for r in results:
         if r.author is None:
             continue
+        for f in r.files:
+            if crabUsername in f.lfn:
+                print f.lfn
```
Comment (blinkseb) [inline scripts/cleanOldFrameworkTasksNotInDB.py]: Can you remove this?

**Comment by OlivierBondu** [comment]
ooops, done

**Comment by blinkseb** [comment]
Looks legit indeed! I'm wondering how relevant is the check on `Sample.path`? We don't use it at all, but I guess it can't hurt?

**Comment by OlivierBondu** [comment]
well it was here in the first place and as you say shouldn't hurt... I don't mind either way

**Pull request merged**